### ISSUE TITLE
Accessibility failing elements with drawer nav categories

### ISSUE THREAD

**Issue by jeffvg** [pxblue/doc-it#108]
#### Describe the bug
accessibility audit found ul.MuiList-root.MuiList-padding.MuiList-subheader and li.MuiListItem-root.MuiListItem-dense.MuiListItem-gutters to be possible issue.
#### What is the expected behavior?
no failing elements
#### What are the steps to reproduce?
1. Clone and run doc'it dev
2. Navigate to new drawer
3. Open dev tools and run audit

#### Screenshots or links to minimum reproduction example
![drawerMuiList](https://user-images.githubusercontent.com/37150565/75062032-f5ea3b80-54af-11ea-94da-c4eec557a76d.png)

#### What is your environment?

 Framework
- [ ] Angular
- [X] React
- [ ] Ionic
- [ ] React Native

 Browser
- [ ] Internet Explorer
- [ ] Edge
- [ ] Safari
- [X] Chrome
- [ ] Firefox

 Operating System
- [ ] MacOS
- [X] Windows
- [ ] Other

**Comment by daileytj** [comment]
Talked with @jeffvg and we decided to close this issue
